js/main.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+var IMAGE_COUNT = 25;
+var MIN_VALUE = 0;
+var AVATAR_MIN_VALUE = 1;
+var AVATAR_MAX_VALUE = 6;
+var LIKES_MIN_VALUE = 15;
+var LIKES_MAX_VALUE = 200;
+var COMMENTS_MIN_PARTS = 1;
+var COMMENTS_MAX_PARTS = 2;
+var COMMENTS_MIN_AMOUT = 1;
+var COMMENTS_MAX_AMOUNT = 3;
+var NAMES = ['Роман', 'Алла', 'Жанна', 'Мирослава', 'Елизавета', 'Максим', 'Наталия', 'Семён', 'Федор', 'Дмитрий', 'Александр', 'Юлия', 'Олег', 'Данил', 'Анастасия', 'Матвей', 'Владислав', 'Наталья', 'Маргарита'];
+var MESSAGES = ['Всё отлично!', 'В целом всё неплохо. Но не всё.', 'Когда вы делаете фотографию, хорошо бы убирать палец из кадра. В конце концов это просто непрофессионально.', 'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.', 'Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.', 'Лица у людей на фотке перекошены, как будто их избивают.', 'Как можно было поймать такой неудачный момент?!'];
+
+var pictureTemplate = document.querySelector('#picture').content.querySelector('.picture');
+var contentGridElement = document.querySelector('.pictures');
+
+var getRandomNumberFromRange = function (min, max) {
+  return Math.floor(Math.random() * (max - min + 1) + min);
+};
+
+var getMessage = function (arr) {
+  var counter = getRandomNumberFromRange(COMMENTS_MIN_PARTS, COMMENTS_MAX_PARTS);
+  var firstMessagePart = getRandomNumberFromRange(0, arr.length - 1);
+
+  if (counter === COMMENTS_MIN_PARTS) {
+    return arr[firstMessagePart];
+  }
+
+  var secondMessagePart = getRandomNumberFromRange(0, arr.length - 1);
+  if (firstMessagePart === secondMessagePart) {
+    return arr[firstMessagePart] + ' ' + arr[(secondMessagePart + 1) % arr.length];
+  }
+  return arr[firstMessagePart] + ' ' + arr[secondMessagePart];
+};
+
+
+var getCommentsValue = function () {
```

Comment:
```suggestion
var getRandomCommentsValue = function () {
```
или
```suggestion
var getSomeCommentsValue = function () {
```
Править не обязательно, но так явно обозначим, что здесь - моковые данные.